.github/workflows/build_and_test.yml

```diff
@@ -32,11 +32,12 @@ jobs:
       style_checker_image_name: "ghcr.io/citusdata/stylechecker"
       style_checker_tools_version: "0.8.18"
       sql_snapshot_pg_version: "17.6"
-      image_suffix: "-va20872f"
+      image_suffix: "-dev-97072ce"
```

Comment:
Needs addressing before landing by merging the branch on the test container repo

Reply:
This one contains pg18 images

will be carefull with it.